client/ignore.lua

```diff
@@ -19,7 +19,7 @@ end)
 CreateThread(function()
     local mapText = Config.PauseMapText
     if mapText == '' or type(mapText) ~= 'string' then mapText = 'FiveM' end
-    Citizen.InvokeNative(joaat('ADD_TEXT_ENTRY'), 'FE_THDR_GTAO', mapText)
+    AddTextEntryByHash('FE_THDR_GTAO', mapText)
```

Comment:
```suggestion
    AddTextEntry('FE_THDR_GTAO', mapText)
```

Why not just use `AddTextEntry` directly / normally?

Reply:
It works, I've tested it several times. "AddTextEntry" seems like the better choice based on its name (and documentation), but I've had occasions where it behaved oddly with blips so it was preference based. Either way, using textentry or textentrybyhash is better than invoking it.